### PR TITLE
Fix typos and grammatical issues

### PR DESCRIPTION
**Pull request by JoeHowse**

### CODE REVIEW

File: update-station [hunk 1]
```diff
@@ -396,8 +396,8 @@ class InstallUpdate:
                     UpdateCompleted()
 
 
-ftxt = """Press 'Detail' to get informations about the failure.
-Get help at https://forums.ghostbsd.org"""
+ftxt = """Press \"Detail\" to get information about the failure.
```
Comment (ericbsd) [inline update-station:399]: Do not need \ in a triple quote.

**Review by ericbsd — CHANGES_REQUESTED** [review]

File: update-station (hunk 1 above)
Comment (JoeHowse) [reply update-station]: Removed "\" in latest commit.

**Review by ericbsd — APPROVED** [review]
LGTM

File: update-station (hunk 1 above)
Comment (ericbsd) [reply update-station]: Thanks

**Pull request merged**